l10n-pocket/en/nav.ftl

```diff
@@ -17,17 +23,24 @@ pocket-nav-highlights = Highlights
 pocket-nav-articles = Articles
 pocket-nav-videos = Videos
 pocket-nav-all-tags = All Tags
-
-# pocket/includes/nav.html
-pocket-nav-open-icon-title = Open
-pocket-nav-open-menu-desc = Open the { -brand-name-pocket } mobile menu
-pocket-nav-logo-text = { -brand-name-pocket }
-pocket-nav-aria-label = Page navigation
-pocket-nav-log-in = Log In
-pocket-nav-sign-up = Sign up
-
-# pocket/includes/mobile-nav.html
 pocket-mobile-nav-aria-label = { -brand-name-pocket } Mobile Menu
 pocket-mobile-nav-close-tooltip = Close
 pocket-mobile-nav-close-menu = Close the { -brand-name-pocket } Mobile Menu
 pocket-mobile-nav-beta = Beta
+
+## Platform navigation is used in some pages:
+## Pocket homepage https://www-dev-pocket.allizom.org
+## Add page URL https://www-dev-pocket.allizom.org/add
+## All platform pages
+## https://www-dev-pocket.allizom.org/ios
+## https://www-dev-pocket.allizom.org/android
+## https://www-dev-pocket.allizom.org/chrome
+## https://www-dev-pocket.allizom.org/safari
+## https://www-dev-pocket.allizom.org/opera
+## https://www-dev-pocket.allizom.org/edge
+
+
+## pocket/includes/platform-nav.html
```

Comment:
Nit: Multi-line group comments should not be separated by blank lines
```suggestion
## pocket/includes/platform-nav.html

```

Reply:
Here too, I separated that line of code because it's supposed to be a group-level comment for the strings that follow it

Reply:
My understanding is that group level comments should be associated with a group of strings. So it you follow one group comment with another group comment, then the first group will no longer have any content associated with it? (the second group comment ends the first). This is why I suggested combining them into one comment.

But perhaps we’re making the multi line comments in this file a little too complicated. Is it worth considering simplifying the amount of information here maybe?

Reply:
Super fair point. I personally thought to place them for developers, so they can see where specific strings are coming from different files, since there are multiple.

However, looking at most other `.ftl` files, the only locations we have placed are the URLs for the pages using the `.ftl` file strings. I can remove them to simplify things. 👍🏼

Reply:
Thinking about developers here is definitely a good thing, however these comments are mostly meant for localizers. Unless we format comments correctly, they likely won't get picked up or be visible by L10n tooling. 

Removing the space underneath `## pocket/includes/platform-nav.html` doesn't change the meaning of the group level comment. I'm suggesting that we have one group level comment for this content group, and not two. Else the first comment will most likely be ignored entirely. Does that make sense?

Reply:
Here's a possible solution that might be close to where you're suggesting (note that the `pocket/includes/platform-nav.html` are no longer group level):

```
## Platform footer is used in some pages:
## Pocket homepage https://dev.tekcopteg.com
## Add page URL https://dev.tekcopteg.com/add
## All platform pages
## https://dev.tekcopteg.com/ios
## https://dev.tekcopteg.com/android
## https://dev.tekcopteg.com/chrome
## https://dev.tekcopteg.com/safari
## https://dev.tekcopteg.com/opera
## https://dev.tekcopteg.com/edge

# pocket/includes/platform-footer.html

pocket-footer-explore = Explore
pocket-footer-publishers = Publishers
pocket-footer-developers = Developers
pocket-footer-support = Support
pocket-footer-jobs = Jobs
```

According to the [Fluent docs](https://projectfluent.org/fluent/guide/comments.html), single hash comments can be stand alone and not associated with only a single string, as long as they are followed by a blank line. This means you would have one group level comment to describe the platform footer strings, and then a single hash comment for each HTML include.

The single hash comment most likely won't get picked up by L10n tools, but it's probably not as relevant for localisers. Perhaps that's the best of both worlds?

Reply:
Agreed. This makes sense. Thanks for pointing out and for being patient with explaining Fluent! I'll add this change.